app/package.json

```diff
@@ -21,62 +21,62 @@
-    "@jupyterlab/application-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/apputils": "~4.0.0-alpha.8",
-    "@jupyterlab/apputils-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/celltags": "~4.0.0-alpha.8",
-    "@jupyterlab/codeeditor": "~4.0.0-alpha.8",
-    "@jupyterlab/codemirror-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/completer": "~4.0.0-alpha.8",
-    "@jupyterlab/completer-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/console": "~4.0.0-alpha.8",
-    "@jupyterlab/console-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/coreutils": "~6.0.0-alpha.8",
-    "@jupyterlab/docmanager": "~4.0.0-alpha.8",
-    "@jupyterlab/docmanager-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/docprovider": "~4.0.0-alpha.8",
-    "@jupyterlab/docprovider-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/documentsearch": "~4.0.0-alpha.8",
-    "@jupyterlab/documentsearch-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/filebrowser": "~4.0.0-alpha.8",
-    "@jupyterlab/filebrowser-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/fileeditor": "~4.0.0-alpha.8",
-    "@jupyterlab/fileeditor-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/hub-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/javascript-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/json-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/mainmenu": "~4.0.0-alpha.8",
-    "@jupyterlab/mainmenu-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/markedparser-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/mathjax2-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/notebook": "~4.0.0-alpha.8",
-    "@jupyterlab/notebook-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/observables": "~5.0.0-alpha.8",
-    "@jupyterlab/outputarea": "~4.0.0-alpha.8",
-    "@jupyterlab/pdf-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/rendermime": "~4.0.0-alpha.8",
-    "@jupyterlab/rendermime-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/rendermime-interfaces": "~4.0.0-alpha.8",
-    "@jupyterlab/running-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/services": "~7.0.0-alpha.8",
-    "@jupyterlab/settingregistry": "~4.0.0-alpha.8",
-    "@jupyterlab/shared-models": "~4.0.0-alpha.8",
-    "@jupyterlab/shortcuts-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/statedb": "~4.0.0-alpha.8",
-    "@jupyterlab/statusbar": "~4.0.0-alpha.8",
-    "@jupyterlab/terminal": "~4.0.0-alpha.8",
-    "@jupyterlab/terminal-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/theme-dark-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/theme-light-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/tooltip": "~4.0.0-alpha.8",
-    "@jupyterlab/tooltip-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/translation": "~4.0.0-alpha.8",
-    "@jupyterlab/translation-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/ui-components": "~4.0.0-alpha.23",
-    "@jupyterlab/user": "~4.0.0-alpha.8",
-    "@jupyterlab/user-extension": "~4.0.0-alpha.8",
-    "@jupyterlab/vega5-extension": "~4.0.0-alpha.8",
+    "@jupyterlab/application": "~4.0.0-alpha.7",
```

Comment:
Looks like these changes were unintentional?

Reply:
They came from the integrity check that ran on push

Reply:
Right the check is pretty simple for now and picks the versions of the locally installed packages (need to fix that).

Reply:
Gotcha, reverted after a git clean.